GCTC-NTGC/gc-digital-talent
# Bug - Fix Classification on summary of filters

Resolves #4500 

## Notes
- I had to add the `selectedClassification` (from the search page filters) to the browser history, since the `PoolCandidateSearchRequest` isn't created till after they submit a request.
- On the other hand, on the admin side. When you view the Request the classification(s) are fetched from pool(s).

## Testing
- Go to Search page, fill out the form and request some candidates.
- Check to see if the Classification has value. If there is no value then request changes and let me know I messed up. If there is a value then fill out the Request form and hit "Submit Request".
- If you are back on the Search page and the mutation succeeded, go to the admin side and view your new request. Now, check  the "Request Information" section to see if the Classification has the same value. If there is no value, request changes and let me know I messed up.
